test/integration/features/addon-xpaas.feature

```diff
@@ -19,18 +19,19 @@ which are then available in OpenShift to the user.
 
   Scenario Outline: User deploys, checks out and deletes several templates from XpaaS imagestream
    Given Minishift has state "Running"
-    When executing "oc new-app <template-name>" succeeds
+    When executing "oc new-project <project-name>" succeeds
+    Then executing "oc new-app <template-name>" succeeds
```

Comment:
I would suggest `And` here as it is still some kind of preparation.
When I deploy app
Then something works